pkg/application/application.go

```diff
@@ -33,13 +34,27 @@ func ListInProject(client *occlient.Client) ([]string, error) {
 		return appNames, nil
 	}
 
+	deploymentSupported, err := client.IsDeploymentConfigSupported()
+	if err != nil {
+		return nil, err
+	}
+
 	// Get all DeploymentConfigs with the "app" label
-	deploymentConfigAppNames, err := client.GetDeploymentConfigLabelValues(applabels.ApplicationLabel, applabels.ApplicationLabel)
+	deploymentAppNames, err := client.GetKubeClient().GetDeploymentLabelValues(applabels.ApplicationLabel, applabels.ApplicationLabel)
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to list applications from deployment config")
```

Comment:
s/unable to list applications from deployment config/unable to list applications from deployment